source/connection-monitoring-and-pooling/tests/pool-checkout-error-closed.yml

```diff
@@ -0,0 +1,23 @@
+version: 1
+style: unit
+description: must throw error if checkOut is called on a closed pool
+operations:
+  - name: checkOut
+    label: conn1
+  - name: checkIn
+    connection: conn1
+  - name: close
+  - name: checkOut
+error:
+  type: PoolClosedError
+  message: Attempted to check out a connection from closed connection pool
+events:
+  - type: ConnectionPoolCreated
+  - type: ConnectionCheckedOut
```

Comment:
`connectionId: 42` should be added to this `ConnectionCheckedOut` event (and the `ConnectionCheckedIn` event below it)